pkg/services/notifications/webhook.go

```diff
@@ -63,6 +64,12 @@ func sendWebRequestSync(ctx context.Context, webhook *Webhook) error {
 		request.Header.Add("Authorization", util.GetBasicAuthHeader(webhook.User, webhook.Password))
 	}
 
+	if len(webhook.HttpHeader) != 0 {
```

Comment:
This if statement can be removed. If webhook.HttpHeader is empty the loop wont do anything

https://play.golang.org/p/Wc_ajoYUfh